README.md

```diff
@@ -319,6 +329,9 @@ sqlboiler -b goose_migrations postgres
 go test ./models
 ```
 
+*Note: No `mysqldump` or `pg_dump` equivalent for Microsoft SQL Server, so generated tests must be supplemented by `tables_schema.sql` with `CREATE TABLE ...` queries*
```

Comment:
Also it is possible to generate schema via SQL Server Management Studio (by Microsoft, goes with SQL Server)

For example:
https://blog.sqlauthority.com/2011/05/07/sql-server-2008-2008-r2-create-script-to-copy-database-schema-and-all-the-objects-data-schema-stored-procedure-functions-triggers-tables-views-constraints-and-all-other-database-objects/